boards/nxp/mimxrt1180_evk/mimxrt1180_evk-pinctrl.dtsi

```diff
@@ -63,4 +63,61 @@
 			input-enable;
 		};
 	};
+
+	pinmux_usdhc1: pinmux_usdhc1 {
+		group0 {
+			pinmux = <&iomuxc_gpio_sd_b1_00_usdhc1_cmd>,
+				<&iomuxc_gpio_sd_b1_01_usdhc1_clk>,
+				<&iomuxc_gpio_sd_b1_02_usdhc1_data0>,
+				<&iomuxc_gpio_sd_b1_03_usdhc1_data1>,
+				<&iomuxc_gpio_sd_b1_04_usdhc1_data2>,
+				<&iomuxc_gpio_sd_b1_05_usdhc1_data3>;
+			bias-pull-up;
+			input-enable;
+		};
+		group1 {
+			pinmux = <&iomuxc_gpio_ad_34_usdhc1_vselect>,
+				<&iomuxc_gpio_ad_15_gpio4_io15>;
+			drive-strength = "high";
+			bias-pull-down;
+			slew-rate = "fast";
+		};
+		group2 {
```

Comment:
Do we need to explicitly MUX the GPIO pins? These should be handled by the GPIO driver in `gpio_configure`, so we shouldn't need to also set the MUX in the pin control setting

Reply:
Sure- Zephyr doesn't generally have that requirement because of the `gpio_configure` API handling it. So let's go ahead and drop the GPIO pins from this list